framework/CHANGELOG.md

```diff
@@ -14,6 +14,7 @@ Yii Framework 2 Change Log
 - Enh #19309: Optimize `yii\base\Model::attributes()` (WinterSilence)
 - Bug #19322: Revert force setting value to empty string in case it's `null` in `yii\validators\FilterValidator::validateAttribute()` (bizley)
 - Bug #19329: Fix `yii\web\GroupUrlRule` to properly normalize prefix (bizley)
+- Enh #19334: Add `$callableValue` for `yii\validators\DefaultValueValidator` (eecjimmy)
```

Comment:
Add `yii\validators\DefaultValueValidator::$callableValue` to callback `$value`

Reply:
i'm sorry form my poor english...:)

Reply:
@eecjimmy other reason: explain how use new property

Reply:
@eecjimmy 
> explain how using new feature

I meaned update `CHANGELOG.md`(this is comments to `CHANGELOG.md`)